engine/src/flutter/display_list/dl_op_records.h

```diff
@@ -595,6 +595,8 @@ DEFINE_DRAW_1ARG_OP(Oval, DlRect, oval)
 DEFINE_DRAW_1ARG_OP(RoundRect, DlRoundRect, rrect)
 #undef DEFINE_DRAW_1ARG_OP
 
+static_assert(sizeof(DrawRoundRectOp) == 56);
```

Comment:
I usually remove these once I am done using them to verify if I'm right. In particular, most of the compilers make the same alignment decisions, but Windows often charts its own course. For now, Windows doesn't run the unit tests that verify these numbers during testing, but it does compile this file and might trip over an unexpected assertion about these numbers.

Reply:
I forgot to remove them after debugging. Thanks for finding them out.